README.md

```diff
@@ -86,6 +85,7 @@ create an app space for Clowdr with any name you like.
    various keys on this page.
 1. Copy `.env-example` as `.env` and update the fields accordingly.
 1. Run `npm install -g parse-server mongodb-runner parse-dashboard`.
+1. Run `cd backend/cloud && npm install`.
 1. Make sure MongoDB is installed on your system.
    Although you do not have to run MongoDB when using Back4App, you will need to
```

Comment:
This looks wrong - for local development, MongoDB does need to be running locally